app/config/errors.php

```diff
@@ -487,6 +487,11 @@
         'description' => 'Index with the requested ID already exists.',
         'code' => 409,
     ],
+    Exception::INDEX_INVALID => [
+        'name' => Exception::INDEX_INVALID,
+        'description' => 'Index invalid.',
```

Comment:
Could we have a more descriptive error message here ?

Reply:
This is only really a placeholder - we pass through the actual error from the validator when we construct the exception, this would only be used in cases we _don't_ do that